src/styles/blocks/card.scss

```
@@ -0,0 +1,111 @@
+.card {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: $card-width;
+  background: hsl(0, 0%, 100%);
+  border: 1px solid hsl(0, 0%, 95%);
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  transition: all 300ms;
```

Do not use all propertie for transition. Set only those properties you change on hover
```suggestion
  transition: transform 0.3s;
```

---

done